### PR TITLE
docs: improve js third party docs

### PR DESCRIPTION
**Pull request by vjerci**
## Description
This pr improves docs for js third party libs

## Related
https://github.com/Bearer/bearer/issues/756

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: pkg/commands/process/settings/rules/javascript/third_parties/airbrake.yml [hunk 1]
```diff
@@ -26,8 +26,18 @@ metadata:
 
     When logging errors or events, ensure all sensitive data is removed.
 
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/airbrake.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/algolia.yml [hunk 2]
```diff
@@ -45,12 +45,19 @@ metadata:
   remediation_message: |
     ## Description
     Leaking sensitive data to third-party data tools is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to Algolia.
-    <!--
+
     ## Remediations
-    Coming soon.
-    -->
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/algolia.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/bugsnag.yml [hunk 3]
```diff
@@ -46,8 +46,18 @@ metadata:
 
     When logging errors or events, ensure all sensitive data is removed.
 
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/bugsnag.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/datadog.yml [hunk 4]
```diff
@@ -32,9 +32,23 @@ metadata:
 
     When logging errors or events, ensure all sensitive data is removed.
 
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/datadog.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/google_analytics.yml [hunk 5]
```diff
@@ -18,10 +20,19 @@ metadata:
 
     When sending data to analytics libraries, ensure all sensitive data is removed.
 
-    <!--
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/google_analytics.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/honeybadger.yml [hunk 6]
```diff
@@ -27,10 +29,18 @@ metadata:
 
     When sending data to logging libraries, ensure all sensitive data is removed.
 
-    <!--
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/honeybadger.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

File: pkg/commands/process/settings/rules/javascript/third_parties/rollbar.yml [hunk 7]
```diff
@@ -29,10 +31,15 @@ metadata:
 
     When sending data to logging libraries, ensure all sensitive data is removed.
 
-    <!--
+    If you really need to identify users, use database's unique identifiers.
```
Comment (elsapet) [inline pkg/commands/process/settings/rules/javascript/third_parties/rollbar.yml]:
```suggestion
    If you really need to identify users, use unique identifiers from the database.
```

**Review by elsapet — APPROVED** [review]
Style suggestion but looks good

**Review by markmichon — APPROVED** [review]

**Pull request merged**